packages/datetime/test/components/timePickerTests.tsx

```diff
@@ -144,16 +144,27 @@ describe("<TimePicker>", () => {
         hourInput.value = "2";
         TestUtils.Simulate.change(hourInput);
         assert.strictEqual(hourInput.value, "2");
+        assert.isFalse(hourInput.classList.contains(CoreClasses.intentClass(Intent.DANGER)));
     });
 
-    it("allows invalid text entry, but shows visual indicator", () => {
+    it("disallows allows non-number text entry", () => {
```

Comment:
I think there's a typo:

```suggestion
    it("disallows non-number text entry", () => {
```

Reply:
Fixed.